dev/com.ibm.ws.webcontainer.security/test/com/ibm/ws/webcontainer/security/WebAppSecurityCollaboratorImplTest.java

```diff
@@ -793,6 +793,8 @@ private WebRequest createWebRequest(String authMethod, final boolean accessPrecl
                 will(returnValue(accessPrecluded));
                 allowing(webRequest).getApplicationName();
                 will(returnValue(APP_NAME));
+                allowing(webRequest).isSSLRequired();
+                will(returnValue(true));
```

Comment:
Does `setHttpServletRequestExpectations` need to be updated as well?  Will `isSecure` be called any longer with your change?